src/App.js

```diff
@@ -21,10 +21,10 @@ export default function App() {
     <>
       <Switch>
         <Route exact path="/">
-          <CropListPage
-            onClickDetails={handleClickDetails}
+          <SearchPage
```

Comment:
what happened to the `CropListPage`? If you don't use it anymore you could delete it.